helfomat-web/src/test/kotlin/de/helfenkannjeder/helfomat/web/configuration/ObjectMapperConfigTest.kt

```diff
@@ -0,0 +1,33 @@
+package de.helfenkannjeder.helfomat.web.configuration
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import de.helfenkannjeder.helfomat.core.ProfileRegistry
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import java.time.LocalTime
+
+@SpringBootTest
+@ExtendWith(SpringExtension::class)
+@ActiveProfiles(ProfileRegistry.TEST)
+class ObjectMapperConfigTest {
+
+    @Autowired
+    private val objectMapper: ObjectMapper? = null
```

Comment:
use `lateinit var` instead of `val`. Then you don't have to assign null and can make it non-optional.